src/layouts/Documentation.vue

```diff
@@ -28,7 +28,7 @@
       <div class="column">
         <slot />
       </div>
-      <div class="column is-3 has-background-white-bis">
```

Comment:
Any reason why this was removed? I think this explains my comment earlier

Reply:
has-background-white-bis forces the background to be white all the time, overriding dark mode stylings. if that's left in, the column has a white background even if dark mode is on.